package.json

```diff
@@ -99,32 +99,23 @@
     "ms.macro": "^2.0.0",
     "multicodec": "^3.0.1",
     "multihashes": "^4.0.2",
-    "node-vibrant": "^3.2.1-alpha.1",
-    "polished": "^3.3.2",
     "polyfill-object.fromentries": "^1.0.1",
     "prettier": "^2.2.1",
     "qs": "^6.9.4",
-    "react": "^17.0.1",
     "react-confetti": "^6.0.0",
     "react-cosmos": "^5.6.3",
-    "react-dom": "^17.0.1",
-    "react-feather": "^2.0.8",
     "react-ga": "^2.5.7",
     "react-is": "^17.0.2",
     "react-markdown": "^4.3.1",
-    "react-popper": "^2.2.3",
     "react-redux": "^7.2.2",
     "react-router-dom": "^5.0.0",
     "react-scripts": "^4.0.3",
     "react-spring": "^8.0.27",
     "react-use-gesture": "^6.0.14",
-    "react-virtualized-auto-sizer": "^1.0.2",
-    "react-window": "^1.8.5",
-    "rebass": "^4.0.7",
     "redux-localstorage-simple": "^2.3.1",
+    "sass": "^1.45.1",
```

Comment:
@zzmp what's this doing here?

Reply:
`microbundle` is not as featureful as I thought, so there is an intermediate asset generation step which (1) generates React components out of svgs and (2) runs sass over our font files: `yarn assets:generate`.

The `sass` step in particular is done so that the font files can be set to use `font-display: block` (I think that `swap` is the default), which prevents the flash of unstyled content (FOUC for google-ability) that would otherwise occur. Note that it's in `devDependencies`, and not `dependencies`.
*Aside: this is done slightly differently in the interface (vs. the widget), which serves a static copy of the font for cache-ability, and is something we can change in the future if need be.*